src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -19,28 +19,34 @@
     display: none;
   }
 
-  .left-controls {
-    float: left;
-    text-align: left;
-    pointer-events: none;
+  .controls-container {
+    width: 100%;
+
+    .left-controls {
+      float: left;
```

Comment:
is there a way to do it with flex ? float can behave unexpectedly.
same for the other uses of float.

Reply:
i try this but it conflicts with other css and break the player in other places, anyway it was already works with float, i want to change only what has to be changed